src/main/scala/org/hammerlab/guacamole/loci/set/TakeLociIterator.scala

```diff
@@ -97,8 +103,9 @@ class TakeLociIterator(it: BufferedIterator[(Position, Coverage)],
    * @return The number of covering regions, and a [[Contig]] with the taken loci ranges.
    */
   def takeLoci(it: ContigIterator[(HasLocus, Coverage)],
-            numRegions: Int,
-            dropAbove: Int): Option[(Int, Contig)] = {
+               numRegions: Int,
+               dropAbove: Int,
+               trimRanges: Boolean): Option[(Int, Contig)] = {
```

Comment:
it was extraneous, this value comes in to the class via the constructor, I've removed it, good catch